src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -34,7 +35,8 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
         m_hinstance(hinstance),
         m_settings(settings),
         m_mouseHook(std::bind(&FancyZones::OnMouseDown, this)),
-        m_windowMoveHandler(settings, &m_mouseHook)
+        m_shiftHook(std::bind(&FancyZones::OnShiftDown, this), std::bind(&FancyZones::OnShiftUp, this)),

```

Comment:
Move `std::bind(&FancyZones::OnShiftUp, this)),` to a new line

Reply:
Do we need to have two separate functions for Up and Down? Wouldn't be better to have one function with an input argument?

Reply:
Good idea, I'll change it now